profiles/migrations/0007_profile_completeness.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.2.4 on 2024-01-04 10:14
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('profiles', '0006_profile_created_at_profile_updated_at'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='profile',
+            name='completeness',
+            field=models.SmallIntegerField(default=0),
+        ),
+    ]
```

Comment:
This action will create the `completeness` with value 0 for all existing profiles.
I think it would be better to add calculation logic in migrations.
Because now all existing profiles on server will have 0 `completeness` until the profile is updated.

https://stackoverflow.com/questions/70588917/django-migrations-calculate-new-fields-value-based-on-old-fields-before-deletin
https://docs.djangoproject.com/en/5.0/ref/migration-operations/

Reply:
Thank you for feedback. Done